tests/101_read_seq_sync_bigfile.fio

```diff
@@ -0,0 +1,8 @@
+; Reads a 4G file in sequential order with 512K block size synchronously. O_DIRECT is turned off in this test.
+; This should work similar to what dd does. (https://github.com/coreutils/coreutils/blob/master/src/dd.c)
+[seq-reader]
+ioengine=sync
+rw=read
+bs=512
+direct=0
+size=4G
```

Comment:
Why `1G` and `4G`? Is this a standard to test or arbitrarily chosen from us? Would like to point to things like that in the README.